docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -105,18 +105,31 @@ When a Global Admin invites a user to join an account, the Global Admin selects
   - Has full administrative permissions across the account, including users and usage
   - Has Namespace Admin [permissions](#namespace-level-permissions) on all [Namespaces](/namespaces) in the account
 - **Developer**
-  - Can create and update Namespaces; has full control over [Workflows](/workflows)
-  - Has Namespace Admin permissions for each Namespace created by that user
-- **Read-Only:** Can only read information
-
-In addition, there are two roles that the Global Admin cannot assign:
+  - Can create Namespaces and modify Namespaces they create
+  - Has Namespace Admin [permissions](#namespace-level-permissions) on all [Namespaces](/namespaces) they've created
+    - Has read-level access to the list of all users within the account so they can add and remove users from a Namespace they've created
+    - Can assign Namespace-level permissions (either read-only or write) to users for a Namespace they've created
+  - Has write permissions for all Namespaces they create
+    - Can control [Workflows](/workflows), [Schedules](/workflows#schedule), etc within those Namespaces
+- **Read-Only (non-administrative)**
+  - Must be assigned to a Namespace by a Namespace Admin
```

Comment:
Do you mean "assigned to a user"? Read-Only access for a namespace is assigned to a user

Reply:
Also this section is to describe the read-only account level role?

In that case they do not have access to any namespaces.

Reply:
Going to discuss on-thread what this means.